### ISSUE TITLE
Releasing Training and Synthetic benchmarks

### ISSUE THREAD

**Issue by mojanjp** [HazyResearch/H3#12]
Hello,
Thanks for the great work! I was wondering if you have plans of releasing the synthetic datasets used in the paper + training scripts for them?

**Comment by DanFu09** [comment]
Yes, we plan to release the synthetics next week. Will post here with an update when we do!

**Comment by ekg** [comment]
Looking forward to this! Hope you get a chance to post these soon!

**Comment by DanFu09** [comment]
These are available here now: https://github.com/HazyResearch/safari
